### PR TITLE
Query Resource Health Alerts

### PR DESCRIPTION
**Pull request by anlucen**
# Overview/Summary

Pull request created for a new query APRL related with Service Healt: Resource Healt and Alert Healt

## Related Issues/Work Items

[AB#29836](https://csusoleng.visualstudio.com/Well-Architected%20Framework/_workitems/edit/29836)

## This PR fixes/adds/changes/removes

1. Resource Healt and Alert Healt query

## As part of this Pull Request I have

- [x] Read the [Contribution Guide](https://azure.github.io/Azure-Proactive-Resiliency-Library/contributing) and ensured this PR is compliant with the guide
- [x] Checked for duplicate [Pull Requests](https://github.com/Azure/Azure-Proactive-Resiliency-Library/pulls)
- [x] Associated it with relevant [GitHub Issues](https://github.com/Azure/Azure-Proactive-Resiliency-Library/issues) or ADO Work Items (Internal Only)
- [x] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Azure-Proactive-Resiliency-Library/tree/main)
- [x] Ensured PR tests are passing
- [x] Performed testing and provided evidence (e.g. screenshot of output) for any changes associated to ARG queries and/or scripts
- [x] Updated relevant and associated documentation (e.g. Contribution Guide, Docs etc.)

### CODE REVIEW

File: docs/content/services/monitoring/resource-health-alerts/code/msr-1.kql
Comment (oZakari) [inline docs/content/services/monitoring/resource-health-alerts/code/msr-1.kql:1]: Could you please update the query to adhere to guidelines mentioned here: https://azure.github.io/Azure-Proactive-Resiliency-Library/contributing/#azure-resource-graph-arg-queries

**Review by oZakari — CHANGES_REQUESTED** [review]
Overall, the PR looks good. I updated a couple of minor things so you'll have to update your local branch as a heads up. Apart from the query changes could you go through and provide the details listed out in the PR form above?

File: docs/content/services/monitoring/resource-health-alerts/code/msr-1.kql
Comment (anlucen) [reply docs/content/services/monitoring/resource-health-alerts/code/msr-1.kql:1]: Hello @oZakari , thank you for the fixing. I added the details i was missing about the description for the query commit and pull request sent

**Comment by oZakari** [comment]
Note from call yesterday: @anlucen to update form: 
image

**Comment by anlucen** [comment]
@oZakari  it was added the description for the pull requested.

**Review by oZakari — APPROVED** [review]

**Review by oZakari — APPROVED** [review]

**Review by ejhenry — APPROVED** [review]
LGTM 👍

**Pull request merged**